rosdep/python.yaml

```
@@ -6768,6 +6768,59 @@ python3-gnupg:
   openembedded: [python3-gnupg@meta-python]
   opensuse: [python3-python-gnupg]
   ubuntu: [python3-gnupg]
+python3-google-auth:
+  debian:
+    '*': [python3-google-auth]
+    stretch:
+      pip:
+        packages: [google-auth]
```

Stretch is currently "oldoldstable" at this point and not supported by the Debian team directly, nor by any active ROS distributions (removed from Melodic in https://github.com/ros-infrastructure/ros_buildfarm_config/pull/188). I would suggest dropping it and eliding the key.
```suggestion
  debian: [python3-google-auth]
```